### PR TITLE
PAAS-2291: split nodes in AZs

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-2291

Short description:

**Don't forget** to update the README and any related documentation if needed: https://confluence.jahia.com/x/bYATAg

### CODE REVIEW

File: packages/jahia/migrations/migrate-to-v26.yaml [hunk 1]
```diff
@@ -111,6 +112,33 @@ actions:
...
+        );
+        if (resp.result != 0)
+          errors.push(resp);
+      });
+      if (errors.length > 0)
+        return {'result': 1, 'errors': errors};
+      else
+        return {'result': 0};
+
+
```
Comment (laurentfufu) [inline packages/jahia/migrations/migrate-to-v26.yaml:140]:
```suggestion
    - environment.nodegroup.ApplyData [cp, bl, sqldb]:
        data:
          distribution:
            mode: SOFT
            zones:
              - az1: zoneA
                az2: zoneB
                az3: zoneC
```

File: packages/jahia/install.yml [hunk 2]
```diff
@@ -66,7 +66,15 @@ onBeforeInstall: |
...
-      ]
+      ],
+    "distribution": {
+        "mode": "SOFT",
+        "zones": [{
+            "az1": "zoneA",
+            "az2": "zoneB",
+            "az3": "zoneC"
+        }]
+      }
```
Comment (laurentfufu) [inline packages/jahia/install.yml:77]: I think using a variable to avoid repetition here would be better

Comment (Ronan-Gorain) [reply packages/jahia/install.yml]: done

File: packages/jahia/migrations/migrate-to-v26.yaml (hunk 1 above)
Comment (Ronan-Gorain) [reply packages/jahia/migrations/migrate-to-v26.yaml]: done

**Review by laurentfufu — APPROVED** [review]

**Pull request merged**